supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -177,7 +177,7 @@
 [discrete]
 [[ceph-object-gateway]]
 ==== image:images/yes.png[yes] Ceph Object Gateway (noun)
-*Description*: In Red Hat Ceph Storage, the Ceph Object Gateway is the S3/Swift component. Also known as RADOS gateway. However, prefer using the Ceph Object Gateway.
+*Description*: In Red Hat Ceph Storage, the _Ceph Object Gateway_ is the S3/Swift component. Also known as _R_ADOS gateway_. However, prefer using the _Ceph Object Gateway_.
```

Comment:
The underscore is in the wrong spot here for `_R_ADOS gateway_`. Also the last instance of "Ceph Object Gateway" should probably be in quotes

Reply:
:heavy_check_mark: